src/github.com/t3chguy/riot-static/register-guest/register-guest.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Michael Telatynski <7t3chguy@gmail.com>
```

Comment:
Go style: Files which generate binaries generally live in `cmd/nameofbinary/main.go`. I'd do the same here.